pom.xml

```diff
@@ -80,6 +80,13 @@
 			<version>${resteasy.version}</version>
 		</dependency>
 		
+		<!-- Libraries -->
```

Comment:
I use Guava mainly for the support with Collections.  Kind of love the Multimap, it's the best thing since List<Object, List<Object>>